cloudbuild.yaml

```diff
@@ -0,0 +1,19 @@
+# steps:
+# - name: "gcr.io/cloud-builders/mvn"
+#   args:
+#   - package
+# artifacts:
+#   objects:
+#   // This is using sources from teh Google Cloud Source Repository
+#     location: "gs://${_BUCKET}/"
+#     paths:
+#     - "target/${_TARGET}"
+
+steps:
+- id: 'Staging the app using mvn appengine plugin'
```

Comment:
Minor note for future reference; the `id` field is intended to be an identifier, not a description.